src/3d/chunks/qgschunkedentity_p.cpp

```diff
@@ -240,69 +245,100 @@ int QgsChunkedEntity::pendingJobsCount() const
   return mChunkLoaderQueue->count() + mActiveJobs.count();
 }
 
-
-void QgsChunkedEntity::update( QgsChunkNode *node, const SceneState &state )
+void QgsChunkedEntity::update( QgsChunkNode *root, const SceneState &state )
 {
-  if ( Qgs3DUtils::isCullable( node->bbox(), state.viewProjectionMatrix ) )
-  {
-    ++mFrustumCulled;
-    return;
-  }
+  QVector<QgsChunkNode *> nodes;
+
+  using slot = std::pair<QgsChunkNode *, float>;
+  auto cmp_funct = []( std::pair<QgsChunkNode *, float> p1, std::pair<QgsChunkNode *, float> p2 )
+  {
+    return p1.second < p2.second;
+  };
+  int renderedCount = 0;
+  std::priority_queue<slot, std::vector<slot>, decltype( cmp_funct )> pq( cmp_funct );
+  pq.push( std::make_pair( root, screenSpaceError( root, state ) ) );
```

Comment:
I was using that at first but using the projected bounding box was problematic because in screen space the bounding box area becomes a 2D polygon in some sense and computing that will be slow (I tried to project each point of the bounding box and compute the bounding box of the resulting polygon in 2D screen space but for some reason that didn't work and was slow too).